index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+  <title>Document</title>
+</head>
+<body>
+  
+  <header>
+    <!-- <img src="./images/background/all_shapes.svg" background="" alt="geometric figures in the background"> -->
+    <div class="header-top">
+      <div class="logo">
+        <p>
+          My Logo
+        </p>
+      </div>
+      <div class="menu">
+        <img src="./images/header/burger_menu.png" alt="menu button">
+      </div>
+    </div>
+    <div class="header-bottom">
```

Comment:
- [ ] nice work. instead of a div please The container of the headline section should be a `section` tag.
* It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.